src/app/core/components/language-picker/language-picker.component.ts

```diff
@@ -13,15 +13,15 @@ export class LanguagePickerComponent implements OnChanges {
   @Output() changeLang = new EventEmitter<string>();
 
   readonly languages = this.localeService.languages;
-  current = this.languages?.find(l => l.tag === this.localeService.currentTag);
+  current = this.localeService.currentLang;
```

Comment:
As the value used by the picker is only the label as the navigation is done only by the locale service now, I think the language picker should only deal with strings and string lists.

Reply:
Fixed